### ISSUE TITLE
rusttype build issues when disabling speedy2D windowing

### ISSUE THREAD

**Issue by MoAlyousef** [QuantumBadger/Speedy2D#8]
Thanks again.
I'm running into a build problem when I disable windowing:
```
error: You need to activate either the `std` or `libm` feature.
  --> /home/rayloom/.cargo/registry/src/github.com-1ecc6299db9ec823/ab_glyph_rasterizer-0.1.4/src/lib.rs:30:1
   |
30 | compile_error!("You need to activate either the `std` or `libm` feature.");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

   Compiling syn v1.0.60
   Compiling owned_ttf_parser v0.6.0
error[E0599]: no method named `sqrt` found for type `f32` in the current scope
  --> /home/rayloom/.cargo/registry/src/github.com-1ecc6299db9ec823/ab_glyph_rasterizer-0.1.4/src/geometry.rs:27:33
   |
27 |         (d.x * d.x + d.y * d.y).sqrt()
   |                                 ^^^^ method not found in `f32`

   Compiling tinyvec v1.1.1
error[E0599]: no method named `abs` found for type `f32` in the current scope
  --> /home/rayloom/.cargo/registry/src/github.com-1ecc6299db9ec823/ab_glyph_rasterizer-0.1.4/src/raster.rs:98:26
   |
98 |         if (p0.y - p1.y).abs() <= core::f32::EPSILON {
   |                          ^^^ method not found in `f32`

error[E0599]: no method named `ceil` found for type `f32` in the current scope
   --> /home/rayloom/.cargo/registry/src/github.com-1ecc6299db9ec823/ab_glyph_rasterizer-0.1.4/src/raster.rs:112:43
    |
112 |         for y in y0..self.height.min(p1.y.ceil() as usize) {
    |                                           ^^^^ method not found in `f32`

error[E0599]: no method named `floor` found for type `f32` in the current scope
   --> /home/rayloom/.cargo/registry/src/github.com-1ecc6299db9ec823/ab_glyph_rasterizer-0.1.4/src/raster.rs:118:30
    |
118 |             let x0floor = x0.floor();
    |                              ^^^^^ method not found in `f32`

error[E0599]: no method named `ceil` found for type `f32` in the current scope
   --> /home/rayloom/.cargo/registry/src/github.com-1ecc6299db9ec823/ab_glyph_rasterizer-0.1.4/src/raster.rs:120:29
    |
120 |             let x1ceil = x1.ceil();
    |                             ^^^^ method not found in `f32`

error[E0599]: no method named `sqrt` found for type `f32` in the current scope
   --> /home/rayloom/.cargo/registry/src/github.com-1ecc6299db9ec823/ab_glyph_rasterizer-0.1.4/src/raster.rs:174:35
    |
174 |         let n = 1 + (tol * devsq).sqrt().sqrt().floor() as usize;
    |                                   ^^^^ method not found in `f32`

error[E0599]: no method named `abs` found for type `f32` in the current scope
   --> /home/rayloom/.cargo/registry/src/github.com-1ecc6299db9ec823/ab_glyph_rasterizer-0.1.4/src/raster.rs:250:32
    |
250 |                 px_fn(idx, acc.abs().min(1.0));
    |                                ^^^ method not found in `f32`

error: aborting due to 8 previous errors

For more information about this error, try `rustc --explain E0599`.
error: could not compile `ab_glyph_rasterizer`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
```
I was able to resolve it by modifying Speedy2D's Cargo.toml:
```toml
[features]
default = ["windowing"]
windowing = ["glutin"]
rusttype-std = ["rusttype/std"]
```
Then using it like so (with fltk):
```toml
[dependencies]
fltk = { version = "0.15", features = ["enable-glwindow"] }
speedy2d = { path = "../speedy2D", default-features = false, features = ["rusttype-std"] }
gl = "0.14"
```
I'm not sure if there's a cleaner way.

**Comment by QuantumBadger** [comment]
Thanks for this -- I've put in a fix [here](https://github.com/QuantumBadger/Speedy2D/commit/dfb71614b5193988fb02f53eabc828d107aaa0da) and published v1.0.2. If you still run into this with the new version please let me know!

**Comment by MoAlyousef** [comment]
Thanks. I confirm it works. I've pushed a demo using fltk and speedy2D here:
https://github.com/MoAlyousef/fltk-rs-demos/tree/master/speedy2d
